pkg/rulefmt/rulefmt.go

```diff
@@ -98,7 +98,7 @@ func (g *RuleGroups) Validate(node ruleGroups) (errs []error) {
 		}
 	}
 
-	return errs
+	return
```

Comment:
There's no defer, so we shouldn't need a named return at all

Reply:
Also we should never have `return` when there are arguments to return, this very prone to errors

Please see https://www.bwplotka.dev/2020/how-thanos-would-program-in-go/#use-named-return-parameters-carefully for rationales.

Reply:
ok

Reply:
@bwplotka Are you happy with this now?